yargs.js

```diff
@@ -907,7 +907,7 @@ function Yargs (processArgs, cwd, parentRequire) {
     }
 
     // register the completion command.
-    completionCommand = cmd || 'completion'
```

Comment:
I'm slightly confused, wouldn't this have defaulted to `completion`? where was the `null` value coming from?

Reply:
Yeah, I'm not adding any custom completion options, I'm wanting to leverage yargs' built-in completions. Because of this I never call the `completion()` function, just `showCompletionScript()`. The completion command will always be `null` unless `completion()` is called (source [1](https://github.com/yargs/yargs/blob/07a5554c480ff3aa43cdc37b1d4ecbaf2687c779/yargs.js#L898), [2](https://github.com/yargs/yargs/blob/07a5554c480ff3aa43cdc37b1d4ecbaf2687c779/yargs.js#L911)).